pkg/apis/cluster/validation/validation.go

```diff
@@ -133,3 +140,38 @@ func ValidateClusterProxyURL(fldPath *field.Path, proxyURL string) field.ErrorLi
 	}
 	return allErrs
 }
+
+// ValidateClusterResourceModels validates cluster's resource models.
+func ValidateClusterResourceModels(fldPath *field.Path, models []api.ResourceModel) *field.Error {
```

Comment:
Please update unit tests.

Reply:
This validate file is not covered by UT.  I will add my method first.